tests/scripts/kubeadm.sh

```diff
@@ -42,13 +42,14 @@ install_master(){
             break
         fi
         echo "."
-        sleep 10
+        sleep 5
         ((++INC))
     done
 
     if [ "${kube_ready}" != "True" ]; then
-        echo "k8s master node never went to Ready status"
-        exit 1
+        echo "k8s master node never went to Ready status. continuing anyway..."
+        kubectl get node -o jsonpath='{.items['$count'].status.conditions[?(@.reason == "KubeletReady")]}'
+        #exit 1
```

Comment:
If we continue, perhaps you should remove this commented line.

Reply:
This was just experimentation to see if the cluster was actually healthy. But when it continued helm failed in the next step, so i still need to figure out the issue with this step.